Maths/test/JugglerSequence.test.js

```diff
@@ -0,0 +1,21 @@
+import { jugglerSequence } from '../JugglerSequence'
+
+describe('Testing isCompositeNumber function', () => {
```

Comment:
Copy-paste remnants: Reads "isCompositeNumber"

Reply:
Hmm, my bad. Should just draft pull request first.